Assets/Scripts/CardDisplay.cs

```diff
@@ -0,0 +1,62 @@
+﻿using UnityEngine;
+using CardGameCode.DataLayer;
+using UnityEngine.UI;
+using TMPro;
+
+namespace CardGameCode.VisualLayer { 
+
+    public class CardDisplay : MonoBehaviour
+    {
+        #region Variables 
+
+        [SerializeField] private TextMeshProUGUI cardDescriptionText, cardStrengthText ,cardNameText;
+        [SerializeField] private Image frontSideImage;
+        [SerializeField] private Scrollbar manaBar;
```

Comment:
Why not to use Slider? It seems much more relevant than Scrollbar to this use case